contracts/mock/MockCoreRefTest.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "../refs/CoreRef.sol";
+
+contract MockCoreRefTest is CoreRef {
```

Comment:
tests should go in test folder and use .t.sol extension

Reply:
this isn't a test though, it's a mock used for a test

Reply:
Just call `MockCoreRef` to remove any confusion?